src/command/persisted_queries/publish.rs

```diff
@@ -68,9 +69,19 @@ impl Publish {
             },
             (Some(_), Some(_), Some(_)) | (Some(_), Some(_), None) | (Some(_), None, Some(_)) => unreachable!("clap \"conflicts_with\" should make this impossible to reach")
         };
+
+        let list_name = name::run(
```

Comment:
Seems like we could run this only in the "list ID provided" case and get this straight from the `resolve` query otherwise? You even added the name to the resolve query but don't use it.

Reply:
oh yeah i was going to do it this way and idk why i didn't. good catch.